src/formatHit.test.js

```diff
@@ -36,6 +36,23 @@ describe('formatHit', () => {
       }
     }),
     getTestCase({
+      name: 'locale_names[1] is matching',
```

Comment:
I just ran the tests on your branch and this is the particular failing test.

Can you double check, use node v6, npm3, rm node_modules and launch tests again?